finishedNumGame.js

```diff
@@ -0,0 +1,43 @@
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+start();
+async function start() { 
+ 
+  console.log("Comp: Let's play a game where you think of a number between 1-100 and I (Comp) try to guess it.\n");
+
+  min = 1  //assign starting minimum
+  max = 100 // assign starting maximum
```

Comment:
You can reassign max to a user input using either `process.argv[2]` or setting the `max = await ask(...)` but remember `await ask(..)` will return a string